ignite/services/plugin/scaffold.go

```diff
@@ -0,0 +1,51 @@
+package plugin
+
+import (
+	"embed"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/gobuffalo/genny"
+	"github.com/gobuffalo/plush"
+	"github.com/gobuffalo/plushgen"
+	"github.com/pkg/errors"
+
+	"github.com/ignite/cli/ignite/pkg/xgenny"
+)
+
+//go:embed template/*
+var fsPluginSource embed.FS
+
+// Scaffold generates a plugin structure under dir/path.Base(moduleName).
+func Scaffold(dir, moduleName string) error {
+	var (
+		name     = filepath.Base(moduleName)
+		finalDir = path.Join(dir, name)
+		g        = genny.New()
+		template = xgenny.NewEmbedWalker(
+			fsPluginSource,
+			"template",
+			finalDir,
+		)
+	)
+	if _, err := os.Stat(finalDir); err == nil {
+		// finalDir already exists, don't overwrite stuff
+		return errors.Errorf("dir %q already exists, abort scaffolding", finalDir)
+	}
+	if err := g.Box(template); err != nil {
+		return errors.WithStack(err)
+	}
+	ctx := plush.NewContext()
+	ctx.Set("ModuleName", moduleName)
+	ctx.Set("Name", name)
+	// plushhelpers.ExtendPlushContext(ctx)
```

Comment:
Will the helper functions added by `plushhelpers.ExtendPlushContext()` eventually be required? If so it would be helpful to have a bit more context in the comment or it they won't be required we could remove the comment.

Reply:
That's definitely not required, I remove it thanks! f3b9a439dd5093ed4638d8ad2b5102e207881450